.github/workflows/verify_semantic_pr.yml

```
@@ -8,6 +8,11 @@ on:
       - synchronize
       - edited
   workflow_call:
+    inputs:
+      runs-on:
+        type: string
+        required: false 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)